script.js

```diff
@@ -1 +1,100 @@
-console.log("Script is working.")
\ No newline at end of file
+console.log("Script is working.")
+
+
+const inputColor = document.querySelector("#input-color")
+const backgroundButton = document.querySelector("#background-button")
+if (inputColor && backgroundButton) {
+    backgroundButton.addEventListener("click", function (){
+        backgroundButton.style.backgroundColor = inputColor.value
+    })
+}
+
+const inputSize = document.querySelector("#input-size")
+const sizeParagraph = document.querySelector("#paragraph-size")
+if (inputSize && sizeParagraph) {
+    inputSize.addEventListener('input', function (){
+        let textSize = Number(inputSize.value)
+        if (5 <= textSize <= 50) {
```

Comment:
what is this construction? did you tested it? confront this approach with two separate conditions (right now numbers 3 and 70 passes your if statement which is not desired isn't it?)